cordova-lib/src/cordova/util.js

```diff
@@ -297,7 +297,7 @@ function ensurePlatformOptionsCompatible (platformOptions) {
     }).map(function (arg) {
         return opts[arg] === true ?
             '--' + arg :
-            '--' + arg + '="' + opts[arg].toString() + '"';
+            '--' + arg + '=' + opts[arg].toString();
```

Comment:
Alright, fair enough. Do you think we can deprecate this usage in the future though? It's a really awkward (and also insecure) API.